assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentToolUtils.java

```diff
@@ -690,6 +768,21 @@ else if ("remove".equals(addUpdateRemoveAssignment)) {
         return alerts;
     } // integrateGradebook
 
+    /**
+     * A utility class to find a gradebook column of a particular name
+     */
+    public org.sakaiproject.service.gradebook.shared.Assignment findGradeBookColumn(String gradebookUid, String assignmentName) {
```

Comment:
Kind of feel that this method should be part of the gradebook service vs here in assignments?

Reply:
try {
   return gradebookService.getAssignment(gradebookUid, assignmentName);
} catch (AssessmentNotFoundException anfe) {
}
return null;